charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```diff
@@ -8,91 +8,100 @@ data:
+        max_connection_idle_time: {{ .Values.config.db.sql.max_connection_idle_time }}
+        max_connection_time: {{ .Values.config.db.sql.max_connection_time }}
+        max_idle_connections: {{ .Values.config.db.sql.max_idle_connections }}
+        max_open_connections: {{ .Values.config.db.sql.max_open_connections }}
+        name: {{ .Values.postgresql.auth.database }}
+        password: {{ .Values.config.db.sql.password }}
+        port: {{ .Values.config.db.sql.port }}
+        replica: {{ .Values.config.db.sql.replica }}
+        skip_initialize_with_version: {{ .Values.config.db.sql.skip_initialize_with_version }}
+        table_prefix: {{ .Values.config.db.sql.table_prefix }}
+        time_zone: {{ .Values.config.db.sql.time_zone }}
+        tx_timeout: {{ .Values.config.db.sql.tx_timeout }}
+        user: {{ .Values.postgresql.auth.username }}
+      sqlite:
+        database_path: {{ .Values.config.db.sqlite.database_path }}
+        debug: {{ .Values.config.db.sqlite.debug }}
+        max_connection_idle_time: {{ .Values.config.db.sqlite.max_connection_idle_time }}
+        max_connection_time: {{ .Values.config.db.sqlite.max_connection_time }}
+        max_idle_connections: {{ .Values.config.db.sqlite.max_idle_connections }}
+        max_open_connections: {{ .Values.config.db.sqlite.max_open_connections }}
+        shared: {{ .Values.config.db.sqlite.shared }}
+    debug: {{ .Values.config.debug }}
+    debug_profiling: {{ .Values.config.debug_profiling }}
+    disable_itc: {{ .Values.config.disable_itc }}
+    arc:
+      url: {{ .Values.config.arc.url }}
+      token: {{ .Values.config.arc.token }}
+      _deployment_id: {{ .Values.config.arc._deployment_id }}
       callback:
-        callback_host: 'https://{{.Values.global.domainName}}'
-      use_fee_quotes: {{ .Values.config.nodes.use_fee_quotes }}
+        enabled: {{ .Values.config.arc.callback.enabled }}
+        host: {{ .Values.config.arc.callback.host }}
+        _token: {{ .Values.config.arc.callback._token }}
+      use_fee_quotes: {{ .Values.config.arc.use_fee_quotes }}
       fee_unit:
-        satoshis: {{ .Values.config.nodes.fee_unit.satoshis }}
-        bytes: {{ .Values.config.nodes.fee_unit.bytes }}
+        satoshis: {{ .Values.config.arc.fee_unit.satoshis }}
+        bytes: {{ .Values.config.arc.fee_unit.bytes }}
     notifications:
       enabled: {{ .Values.config.notifications.enabled }}
       webhook_endpoint: "{{ .Values.config.notifications.webhook_endpoint }}"
+    block_headers_service:
+      auth_token: {{ .Values.config.block_headers_service.auth_token }}
+      url: {{ .Values.config.block_headers_service.url }}
     paymail:
       beef:
-        {{- if .Values.config.paymail.beef.block_headers_service_url }}
+        block_headers_service_auth_token: {{ .Values.config.paymail.beef.block_headers_service_auth_token }}
         block_headers_service_url: {{ .Values.config.paymail.beef.block_headers_service_url }}
```

Comment:
we should calculate block_headers_service_url.
 If the `.Values.config.block_headers_service.url` is set, then use that value + `/api/v1/chain/merkleroot/verify` suffix, 
if not, then calculate the value as `'http://{{- printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-" }}/api/v1/chain/merkleroot/verify'`

Notice that this value is sligthly different because it needs to have also this suffix `/api/v1/chain/merkleroot/verify`